src/core/simulation/simulation.ts

```diff
@@ -122,15 +122,104 @@ export class Simulation extends Config {
       stepSize: 1,
     };
   }
+  /**
+   * Run NEST simulation in CoSim.
+   *
+   * @remarks
+   * After the simulation it updates the activities and commits the network.
+   */
+  private async runCoSimulation(): Promise<any> {
+    this.consoleLog('Run Co-simulation');
+
+    return this.backends.coSim.instance
+      .post('write', {
+        script: this._code.script
```

Comment:
Prettier suggestion:
```suggestion
        script: this._code.script,
```